contracts/tests/SpigotedLine.t.sol

```diff
@@ -1088,5 +1088,38 @@ contract SpigotedLineTest is Test {
       vm.prank(lender);
       line.claimAndRepay(address(revenueToken), tradeData);
     }
+
+    // tests that the amount used to repay the lender cannot cause an underflow
+    function test_lender_use_and_repay_underflow() public {
```

Comment:
Can u add another test making sure we cant call useAndRepay with a token that doesnt exist (check we dont have one already)

Reply:
`useAndRepay` only takes an amount